src/main/java/org/junit/runners/model/FrameworkField.java

```diff
@@ -15,7 +14,7 @@
 public class FrameworkField extends FrameworkMember<FrameworkField> {
     private final Field field;
 
-    FrameworkField(Field field) {
+    public FrameworkField(Field field) {
```

Comment:
Let's add JavaDoc and indicate that this constructor is public as of JUnit 4.14.

Reply:
@pholser you somehow lost the "change to public" part of the change :-)

I added a commit with suggested edits. If you agree, please squash with the second commit.